custom_model_runner/datarobot_drum/drum/args_parser.py

```diff
@@ -330,11 +330,26 @@ def _reg_arg_docker(*parsers):
                 required=False,
                 help="Docker image to use to run {} in the {} mode, "
                 "or a directory, containing a Dockerfile, "
-                "which can be built into a docker image. ".format(
-                    ArgumentsOptions.MAIN_COMMAND, prog_name_lst[1]
+                "which can be built into a docker image. "
+                "If code dir contains requirements.txt file, DRUM tries to install dependencies during image build. "
```

Comment:
note in description this is meant to parallel the requirements.txt functionality in the app.

Reply:
DONE